src/intl/messages.js

```diff
@@ -43,6 +43,7 @@ export const messages: { [messageId: string]: MessageType } = {
   bootSequence: 'Boot Sequence',
   bootSequenceTooltip: 'Device order in which VM will search OS for running.',
   cancel: 'Cancel',
+  cantLogonToConsole: 'Can\'t logon to console. Do you want to continue?',
```

Comment:
Perhaps a bit more descriptive:

"SSO logon to the VM's console failed.  Do you want to continue without SSO?"